code/obj/flock/furniture.dm

```diff
@@ -34,6 +34,18 @@
 		<br><span class='bold'>ID:</span> Storage Surface
 		<br><span class='bold'>###=-</span></span>"}
 
+/obj/table/flock/Crossed(atom/movable/mover)
+	. = ..()
+	var/mob/living/critter/flock/drone/drone = mover
+	if(istype(drone) && !drone.floorrunning)
+		animate_flock_passthrough(mover)
+		. = TRUE
+	else if(istype(mover,/mob/living/critter/flock))
+		. = TRUE
+
+/obj/table/flock/Cross(atom/movable/mover)
+	return istype(mover,/mob/living/critter/flock)
+
```

Comment:
Do drones really need to pass through tables?

Reply:
it mostly just makes them consistent with other flock things (buildings, barricades, etc). I did it as a result of play testing, and I feel like it improved things.

Reply:
It shouldn't need to be added, `jpsTurfPassable()` checks for `Cross()` on all objects in a turf, so just adding `Cross() return istype(flock)` is sufficient to make any item pathable through for flock critters.